Octokit/Models/Response/PullRequestReviewComment.cs

```diff
@@ -15,7 +15,7 @@ public PullRequestReviewComment(int id)
             Id = id;
         }
 
-        public PullRequestReviewComment(string url, int id, string diffHunk, string path, int? position, int? originalPosition, string commitId, string originalCommitId, User user, string body, DateTimeOffset createdAt, DateTimeOffset updatedAt, string htmlUrl, string pullRequestUrl)
+        public PullRequestReviewComment(string url, int id, string diffHunk, string path, int? position, int? originalPosition, string commitId, string originalCommitId, User user, string body, DateTimeOffset createdAt, DateTimeOffset updatedAt, string htmlUrl, string pullRequestUrl, int? inReplyToId)
```

Comment:
Looks like you forgot to assign the passed in variable to the class member, within the constructor

Reply:
That's embarrassing. Fixed in latest commit.